tests/functional/kill.bats

```diff
@@ -144,3 +121,35 @@ function teardown() {
 	run_cmd "$cmd" "0" "$COR_TIMEOUT"
 	verify_runtime_dirs "$container_id" "deleted"
 }
+
+@test "start then kill --all" {
+	signal=SIGTERM
+	trap_file=/trap.log
+	workload_cmd \
+		"sh" "-c" \
+		"trap 'touch ${trap_file} ; exit 0' $signal \&\& while : ; do sleep 1; done"
```

Comment:
Yeah, I know. I wasn't suggesting you change it - seeing the backslashed aphersands just made me laugh. Maybe I need to get out more ;-)